docker.sh

```diff
@@ -1,2 +1,10 @@
 #!/bin/bash
-docker container run --rm -it -v "$(pwd)":/full-fledged-hledger dastapov/full-fledged-hledger:latest
```

Comment:
In Microsoft WSL this doesn't work because we need to use Windows paths instead of Linux ones.